tgui/packages/tgui/interfaces/PaperSheet.js

```diff
@@ -235,26 +225,29 @@ const setInputReadonly = (text, readonly) => {
 
 // got to make this a full component if we
 // want to control updates
-const PaperSheetView = (props, context) => {
+export const PaperSheetView = (props, context) => {
   const {
-    value,
-    stamps,
+    value = "",
+    stamps = [],
     backgroundColor,
     readOnly,
   } = props;
-  const stamp_list = stamps || [];
+  const stamp_list = stamps;
   const text_html = {
     __html: '<span class="paper-text">'
       + setInputReadonly(value, readOnly)
       + '</span>',
   };
   return (
     <Box
+      className="paper__page"
       position="relative"
       backgroundColor={backgroundColor}
-      width="100%"
-      height="100%" >
+      width={"100%"}
+      height={"100%"} >
       <Box
+        className="Paper__Page"
```

Comment:
Box in a Box with *very* similar className is weird here. I also am unclear on what class this one is supposed to be (should these be lowercased?)

(I fully suspect this is something to do with how elements are positioned re: relative/absolute/etc., but would like it looked into a little if possible.)